packages/data-portal-utils/src/lib/types.ts

```diff
@@ -7,6 +7,7 @@ export interface IAttributeInfo<T> {
 
 export enum AttributeNames {
     // Synapse attribute names
+    OrganType = 'OrganType',
```

Comment:
we should probably move this below the `// Derived or attached in frontend` comment since this is not an actual Synapse attribute name.

Reply:
Done. Also, I have used the lowercase for the first letter for the Attribute Name to be consistent with existing Derived/attached attribute names.